admin-ui/src/pages/settings-page.tsx

```
@@ -266,6 +293,315 @@ function aliasRecordFromItems(items: Array<ModelAliasItem>): ModelAliasRecord {
   return record
 }
 
+const BLOCKED_PROVIDER_KEYS = new Set(["__proto__", "constructor", "prototype"])
+
+function providerItemsFromRecord(record: ProviderRecord | undefined): Array<ProviderItem> {
```

ESLint

[error] Replace `record:·ProviderRecord·|·undefined` with `⏎··record:·ProviderRecord·|·undefined,⏎`

(prettier/prettier)